homeassistant/components/thermostat/zwave.py

```diff
@@ -52,6 +52,10 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                 _LOGGER.debug("Remotec ZXT-120 Zwave Thermostat, ignoring")
                 return
 
+    if value.command_class != COMMAND_CLASS_SENSOR_MULTILEVEL and \
+       value.command_class != COMMAND_CLASS_THERMOSTAT_SETPOINT:
+        return
```

Comment:
I see I need to figure out a different method to do this. Because we pass in only one command_class to begin with.